package.json

```diff
@@ -28,16 +28,16 @@
     "@skyux/grids": "^3.0.0",
     "@skyux/i18n": "^3.5.0",
     "@skyux/indicators": "^3.0.0",
-    "@skyux/list-builder": "^3.0.1",
-    "@skyux/list-builder-view-checklist": "^3.0.0",
+    "@skyux/list-builder": "^3.2.1",
+    "@skyux/list-builder-view-checklist": "^3.1.0",
```

Comment:
Just confirming that both of these peer changes are required?

Reply:
List builder - absolutely. It has all the nuts & bolts for multiselect to work. Checklist, on the other hand, wouldn't necessarily need updated here. I'll revert it back.

Reply:
yep, it works fine. reverted.